docker/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#  Copyright (C) 2018-2019 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+sudo cp -r /opt/autoware/src/autoware.ai/ros/src/msgs/autoware_msgs ~/src/autoware_msgs
```

Comment:
`mv` would be preferable to `cp` unless the rest of the autoware codebase is also being built, slightly reduces image size.